src/userdev/java/net/minecraftforge/gradle/userdev/util/Deobfuscator.java

```diff
@@ -158,7 +158,7 @@ public File deobfSources(File original, String mappings, String... cachePath) th
         if (!cache.isSame() || !output.exists()) {
             McpNames map = McpNames.load(names);
 
-            try (ZipInputStream zin = new ZipInputStream(new FileInputStream(input));
+            try (ZipInputStream zin = new ZipInputStream(new FileInputStream(original));
```

Comment:
Why? Add a comment why this change is needed. As far as I can see this change is not needed.

Reply:
The input file is not exist when this line is reached. It is broken and I have no idea why previous contributor would write this.

Reply:
Okey Perfect resolves the issue.